CPT331.Core/ObjectModel/SessionKey.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CPT331.Core.ObjectModel
+{
+    /// <summary>
+    /// Encapsulates the session key used for the MVC site authentication. 
+    /// </summary>
+    public static class SessionKey
+    {
+        /// <summary>
+        /// This key is for an admin user session. It's used by the Post 
+        /// Login action method and the AdminAutorizeAttribute class. 
+        /// </summary>
+        public static string Key { get { return _userSessionKey; } }
+        private const string _userSessionKey = "User";
+    }
+}
```

Comment:
I think it would be awesome to create some abstraction around the HttpContext.Current.Session variable it self. 

So rather than accessing the user with the session variable, you might use ```SessionKey.UserName != null``` or better yet, create another property ```SessionKey.LoggedIn``` that handles the implementation.

Maybe this is something to tackle in a future PR.